src/EventManagement.Services/Pdf/NodePdfRenderService.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.NodeServices;
+
+namespace losol.EventManagement.Services.Pdf
+{
+    [Obsolete("Use Converto instead")]
+    public class NodePdfRenderService : IPdfRenderService
```

Comment:
Thanks for making this proper, however I think we could just trash the old NodePdfRenderService.

Reply:
Sure, but it's just a good practice - to isolate external deps this way. So they could be replaced easily. Would remove it in the separate pr. Because, who knows maybe some test cases would fail and you will still have the option for it to work.

Reply:
Ok. I agree, thanks!